client/query.go

```diff
@@ -52,19 +52,6 @@ func (ctx Context) QueryABCI(req abci.RequestQuery) (abci.ResponseQuery, error)
 	return ctx.queryABCI(req)
 }
 
-// QuerySubspace performs a query to a Tendermint node with the provided
-// store name and subspace. It returns key value pair and height of the query
-// upon success or an error if the query fails.
-func (ctx Context) QuerySubspace(subspace []byte, storeName string) (res []sdk.KVPair, height int64, err error) {
-	resRaw, height, err := ctx.queryStore(subspace, storeName, "subspace")
-	if err != nil {
-		return res, height, err
-	}
-
-	ctx.LegacyAmino.MustUnmarshalBinaryBare(resRaw, &res)
```

Comment:
I fmt.Printlned the `res` here, it's a `[]KVPair` of `[]bytes->[]bytes`, and I'm assuming those bytes are proto-encoded.

This means that the client.Context would need to hold a reference to the BinaryMarshaler too, which seems like a code smell.

Instead, I deleted this function (only used in 1 place), and used gRPC queries in GetCmdQueryValidators.

Reply:
So, I assume we are.ok to delete this public function?